package.json

```diff
@@ -18,8 +18,8 @@
     "clean": "lerna run clean && lerna clean --yes",
     "build": "lerna run build --concurrency=1 --stream",
     "build:prod": "npm run update-deps && lerna run build:prod --concurrency=1 --stream",
-    "lint": "eslint 'packages/*/src/**/*.ts'",
-    "lint-fix": "eslint 'packages/*/src/**/*.ts' --fix",
+    "lint": "eslint 'packages/*/src/**/!(*.d).ts'",
```

Comment:
We should also lint declaration files for third-party modules, if they're included in the project.